dojo/tools/h1/parser.py

```diff
@@ -29,37 +29,49 @@ def get_findings(self, file, test):
         # Load the contents of the JSON file into a dictionary
         data = file.read()
         try:
-            tree = json.loads(str(data, 'utf-8'))
-        except:
+            tree = json.loads(str(data, "utf-8"))
+        except BaseException:
             tree = json.loads(data)
         # Convert JSON  report to DefectDojo format
         dupes = dict()
         for content in tree["data"]:
             # Get all relevant data
-            date = content['attributes']['created_at']
-            date = datetime.strftime(datetime.strptime(date, "%Y-%m-%dT%H:%M:%S.%fZ"), "%Y-%m-%d")
+            date = content["attributes"]["created_at"]
+            date = datetime.strftime(
+                datetime.strptime(date, "%Y-%m-%dT%H:%M:%S.%fZ"), "%Y-%m-%d"
+            )
             # Build the title of the Dojo finding
             title = "#" + content["id"] + " " + content["attributes"]["title"]
 
             description = self.build_description(content)
 
             # References
             try:
-                issue_tracker_id = content['attributes']['issue_tracker_reference_id']
-                issue_tracker_url = content['attributes']['issue_tracker_reference_url']
-                references = "[{}]({})\n".format(issue_tracker_id, issue_tracker_url)
-            except:
+                issue_tracker_id = content["attributes"][
+                    "issue_tracker_reference_id"
+                ]
+                issue_tracker_url = content["attributes"][
+                    "issue_tracker_reference_url"
+                ]
+                references = "[{}]({})\n".format(
+                    issue_tracker_id, issue_tracker_url
+                )
+            except BaseException:
```

Comment:
```suggestion
            except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50